SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -817,31 +777,31 @@ void FSpatialGDKEditorToolbarModule::VerifyAndStartDeployment(FString ForceSnaps
 
 		FSpatialLaunchConfigDescription LaunchConfigDescription = SpatialGDKEditorSettings->LaunchConfigDesc;
 
-		FWorkerTypeLaunchSection Conf = SpatialGDKEditorSettings->LaunchConfigDesc.ServerWorkerConfig;
 		// Force manual connection to true as this is the config for PIE.
-		Conf.bManualWorkerConnectionOnly = true;
-		if (Conf.bAutoNumEditorInstances)
+		LaunchConfigDescription.ServerWorkerConfiguration.bManualWorkerConnectionOnly = true;
```

Comment:
Can't this now be forced inside the GenerateLaunchConfig function? As it now knows if it's local or cloud.
But I can be convinced either way - the GenerateLaunchConfig can be a "purer" function by only literally respecting whatever config is passed to it.

Reply:
So this is legacy from the old implementation, I'd rather keep it the same for now